templates/CRM/Sumfields/Form/SumFields.tpl

```diff
@@ -33,7 +33,12 @@
 </table>
 
 <h3>{ts}Field Settings{/ts}</h3>
-
+<div>
+  <span class="label">{$form.show_simplified.label}</span>
+  <span>{$form.show_simplified.html}</span>
+  <div class="description">{ts}Shows a simplified field definition by using the table <strong>civicrm_contribution</strong> instead of <strong>civicrm_line_item</strong>.{/ts}</div>
+  <div class="description">{ts}Please note: If you check/uncheck this checkbox, all previous selected fields will be <strong>unchecked!</strong>.{/ts}</div>
+</div>
 {foreach from=$fieldsets key="title" item="fields"}
```

Comment:
This is really hard to succinctly explain! Maybe line 39 could be replaced with:

Use simplified contribution fields. By default, contribution fields are calculated using the line items table, which provides the most accurate accounting if you use price sets with different financial types. Check this box to use simplified contribution fields instead. Simplified contribution fields are calculated using the contribution table, which is more efficient and will work better on large installations but won't accurately count a single contribution split between two line items (e.g. an event registration and donation).

Too long??

Reply:
I would certainly prefer to have it displayed as you wrote it. At least it explains in depth what's that button for and it's not cryptic at all.